cli/listener/src/main.rs

```diff
@@ -36,24 +40,13 @@ pub struct Opts {
     #[clap(long, default_value = "300")]
     poll_frequency_seconds: u32,
 
-    /// Location of the SQLite DB file.
-    #[clap(long, default_value = "listener.db")]
+    /// Location of the SQLite DB file, defaults to "listener.db".
```

Comment:
Clap will already append the default, we shouldn’t mention it in the doc comment. Right now it prints this:

```
        --db-path <db-path>
            Location of the SQLite DB file, defaults to "listener.db" [default: listener.sqlite3]
```